TouchPortalAPI/tools.py

```diff
@@ -19,6 +19,37 @@
 import requests
 import os
 import base64
+from types import SimpleNamespace
+import logging
+import sys
+
+class Log(logging.Logger):
```

Comment:
This class should be removed from this PR because it will cause conflicts once the logging PR is ready to merge.  Also the related import.

Reply:
Forgot to remove that